src/components/SummaryList.tsx

```diff
@@ -0,0 +1,59 @@
+import classNames from "classnames";
+import PropTypes, { arrayOf } from "prop-types";
+import React from "react";
+
+import "govuk-frontend/govuk/components/summary-list/_summary-list.scss";
```

Comment:
This could do with a comment pointing out that it comes from `govuk-frontend` rather than `lbh-frontend`.